### PR TITLE
Version-bump-v2.7.4

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
= 2.7.4 - TUESDAY, 8th AUGUST 2023 =
* Improvement: Core Blocks - Implemented responsive typography font-size units.
* Improvement: Image - Added a new `Custom Size` feature, which will maintain the dimensions set when the image is replaced!
* Improvement: Popup Builder - Moved page-level settings into block-level panels.
* Improvement: Testimonial - Added a vertical alignment option for equal-height content.
* Improvement: Global - Added a fresh new look for preset icons and rearranged some settings.
* Fix: Forms - Resolved an issue wherein the submit button would not take the padding set in the customizer.
* Fix: Call To Action - Resolved an issue wherein the line-height of the title would not appear responsive in the frontend.
* Fix: Inline Notice - Updated inconsistent default padding and removed unwanted top-margin in the editor and the frontend.
* Fix: Global - Resolved an issue in which the spacing between the text and the icon in buttons would not be applied as intended in RTL mode for certain blocks.
* Fix: Testimonial - Resolved an issue where there is extra margin for content in mobile view.

- [x]  Run to update version number : `grunt bump-version --ver=[version-number]`
- [x]  Verify `Stable tag` is [version-number] in readme.txt
- [x]  Verify `Tested upto` is set to latest tested version of WordPress
- [x]  Verify changelog `date` and `content` as per Spectra standards
- [x]  Generate README.md : `grunt readme`
- [x]  Update `version` in `ultimate-addons-for-gutenberg.php` in plugin description
- [x]  Verify constant `UAGB_VER` in `class-uagb-loader.php` with latest version of Spectra
- [x]  Generate POT file : `npm run makepot`
- [x]  Update version in `package.json `and `package-lock.json`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ultimate-addons-for-gutenberg",
-  "version": "2.7.3",
+  "version": "v2.7.4",
```
Comment (adi3890) [inline package.json:3]: it should not be v2.7.4
please make it 2.7.4

Comment (adi3890) [reply package.json:3]: check it everywhere

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 **Requires at least:** 5.6  
 **Requires PHP:** 5.6  
 **Tested up to:** 6.2.2
```
Comment (adi3890) [inline README.md:7]: tested upto needs to be 6.3.0

File: package.json (hunk 1 above)
Comment (rajmohanreddy774) [reply package.json]: Updated.

File: README.md (hunk 2 above)
Comment (rajmohanreddy774) [reply README.md]: Changed.

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md
Comment (bsf-zanev) [inline README.md:1]: Changelog needs to be rearranged like this:

Improvements are correct.
Fixes need to be rearranged and slightly updated.

```
* Fix: Call To Action - Resolved an issue wherein the line-height of the title would not appear responsive in the frontend.
* Fix: Forms - Resolved an issue wherein the submit button would not utilize the padding set in the customizer.
* Fix: Inline Notice - Updated inconsistent default padding and removed unwanted top-margin in the editor and the frontend.
* Fix: Testimonial - Resolved an issue in which an extra margin would appear for the content in the mobile view.
* Fix: Global - Resolved an issue in which the spacing between the text and the icon in buttons would not be applied as intended in RTL mode for certain blocks.
```

Comment (bsf-zanev) [inline README.md:1]: Add an upgrade notice like this:

```
2.7.4
With WordPress 6.3 no longer supporting older versions of PHP, the next Spectra release will also require a minimum of PHP 7.4. For websites running on older version of PHP, we strongly recommend upgrading to PHP 7.4 or above to ensure optimal performance and compatibility.
```

Comment (rajmohanreddy774) [reply README.md:1]: Done.

Comment (rajmohanreddy774) [reply README.md:1]: Added.

File: readme.txt [hunk 3]
```diff
@@ -1525,3 +1525,6 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 = 2.7.3 =
 We've optimized the internal structure of Spectra in this update. To ensure your website's safety, We recommend that you backup your website before updating.
+
+= 2.7.4 =
```
Comment (adi3890) [inline readme.txt:1529]: This should be above 2.7.3 as we order for changelog

**Review by bsf-zanev — APPROVED** [review]

File: readme.txt (hunk 3 above)
Comment (rajmohanreddy774) [reply readme.txt:1526]: Updated.

File: README.md [hunk 4]
```diff
@@ -1525,3 +1525,6 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ### 2.7.3 ###
 We've optimized the internal structure of Spectra in this update. To ensure your website's safety, We recommend that you backup your website before updating.
+
+### 2.7.4 ###
+With WordPress 6.3 no longer supporting older versions of PHP, the next Spectra release will also require a minimum of PHP 7.4. For websites running on older version of PHP, we strongly recommend upgrading to PHP 7.4 or above to ensure optimal performance and compatibility.
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply README.md]: Updated.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**